test/specs/initial_spec.js

```diff
@@ -1,28 +1,82 @@
-import swarmAnimation from '../../src/js/swarmAnimation';
+import { swarmAnimation, DURATION_TYPE } from '../../src/js/swarmAnimation';
 
 describe('a first test boilerplate for swarmAnimation functions', function () {
 
-	let el;
+	jasmine.getStyleFixtures().fixturesPath = '../../dist';
 
-	beforeEach(() => {
-		el = document.createElement('div');
-		document.body.appendChild(el);
-	});
+	describe('opacity and fade tests', function() {
 
-	afterEach(() => {
-		el.remove();
-		el = null;
-	});
+		const opacityClass0 = 'anim-opacity--0';
+		const opacityClass1 = 'anim-opacity--1';
+		const transitionClass = 'anim-transition--opacity';
 
-	it('adds one to a given number (dummy test)', function () {
-		expect(swarmAnimation.addOneTest(1)).toEqual(2);
-	});
+		describe('opacity classes', function() {
+
+			let el;
+
+			beforeEach(() => {
+				loadStyleFixtures('animation.css'); // eslint-disable-line no-undef
+				el = document.createElement('div');
+				document.body.appendChild(el);
+			});
+
+			afterEach(() => {
+				el.remove();
+				el = null;
+			});
+
+			it('class gives proper opacity 0', function () {
+				el.className = opacityClass0;
+				const opacity = window.getComputedStyle(el).getPropertyValue('opacity');
```

Comment:
I'm using jasmine not jest. 
It was a little trial and error because it seemed like only the first opacity test would pass. It seems you need to load the fixtures before each test. What you can't do is test actual animation completion, transitionEnd events etc.